group-generators/helpers/data-providers/farcaster/index.ts

```diff
@@ -0,0 +1,95 @@
+import axios from "axios";
+import { retryRequest } from "@group-generators/helpers/data-providers/utils/utils";
+import { FetchedData } from "topics/group";
+
+export class FarcasterProvider {
+  url: string;
+  headers: {
+    authorization: string;
+    accept: string;
+  };
+
+  public constructor() {
+    this.url = "https://api.farcaster.xyz/v2/";
+    this.headers = {
+      authorization: process.env.FARCASTER_API_KEY as string,
```

Comment:
You need to have the "Bearer" outside of the `FARCASTER_API_KEY`. 
Is it the case?

Reply:
It works without bearer, nevermind

Reply:
No the "Bearer" is part of the the `FARCASTER_API_KEY`